### DISCUSSION TITLE
Style update for 'new look'/theme in MDAs?

### DISCUSSION THREAD

**Discussion by wenstarner** [khoait/DCE.PCF#83] [General] [answered]
Hi, this is an amazing PCF component. I think in the Top 5 of all time. Thank you so much for this!

I was wondering whether you might consider updating the styling to match the 'new look'/theming for model-driven apps? Like in the screenshot below. I think that would take it to the next level. 

Anyway, thanks again for contributing this amazing component to the community! Much appreciated 😊

<img width="394" alt="image" src="https://github.com/khoait/DCE.PCF/assets/127121877/fe8bd8d6-f692-4739-9cf7-1e72079d2266">

**Comment by khoait** [comment, marked as answer]
Hi @wenstarner thanks for your feedback!

As the Fluent2 has gone GA in Model Driven App, I agree the components should apply the new look and feel. However, I'll need to think about the right time to make the change. After trying out the new UI, I find it incompleted in some cases, also users still have the option to switch the UI back and forth. That makes supporting both UIs difficult, the components become heavy and hard to maintain.

Maybe I'll create separate components for the new UI or we should wait a little bit until the 2024 Release Wave 1 when everything more stable. I'll create an issue and put it in the backlog anyway.

Please consider giving the repo a star if you find it useful 😄

**Reply by wenstarner** [reply]
Thanks so much for the reply! And that all makes sense... Will give your repo a star. 

I'm looking forward to seeing what else you develop! :)
